### ISSUE TITLE
Is it possible to send the pipeline status messages to different channel for Master?

### ISSUE THREAD

**Issue by pazt** [jenkins-x-labs/issues#22]
Hi,

Thank you for publishing this awesome project !!!
We have installed it and we wonder if we can send pipeline messages of pull requests to one Slack channel and the pipeline Master release to a different channel ?

If it possible, How can I configure it?
My conf today look like (so PR and Master pipeline steps messages going to the same channel now):

spec:
  namespace: jx
  pipelines:
  - directMessage: false
    orgs:
    - name: hibobio
  - channel: jx-hibob-web-client
    notifyReviewers: true
    orgs:
    - name: hibobio
      repos:
      - hibob-web-client
  - channel: jx-hibob
    notifyReviewers: true
    orgs:
    - name: hibobio
      repos:
      - hibob
  - channel: jx-staging
    notifyReviewers: true
    orgs:
    - name: hibobio
      repos:
      - environment-hibob-staging
  - channel: jx-production
    notifyReviewers: true
    orgs:
    - name: hibobio
      repos:
      - environment-hibob-production
  tokenReference:
    kind: Secret
    name: slack-bot-secret

**Comment by rawlingsj** [comment]
I don't believe it's supported at the moment, related to this issue https://github.com/jenkins-x-labs/slack/issues/20

**Comment by pazt** [comment]
Thank you for replying.
Is the future feature "Can be configured to send failure messages when release pipelines fail" will be implemented soon?

**Comment by rawlingsj** [comment]
ATM my focus is towards JX3 so I wont be able to take a look for sometime until we get to GA.  That's not to say someone couldn't have a go themselves and contribute it, if that's something your interested in I could familiarise myself with the code and try to give some pointers?

FWIW there is some tooling in Jenkins X that helps know about failed pipelines.  The Jenkins X plugin for Octant has a view that shows the result of pipelines.  I know it's not what you're after here but might help in the meantime.  

https://github.com/jenkins-x/octant-jx/

<img width="834" alt="Screenshot 2020-08-04 at 09 59 32" src="https://user-images.githubusercontent.com/1457180/89274830-5d004480-d639-11ea-9ac6-70572575d67b.png">
